debian/marquee/nginx/sites/n.whatwg.org.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name n.whatwg.org;
+    root /var/www/n.whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/n.whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/n.whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
+
+    location / {
+        autoindex on;
+        autoindex_localtime off;
+        autoindex_exact_size off;
+    }
+
+    rewrite /work https://html.spec.whatwg.org/multipage/microdata.html#licensing-works;
```

Comment:
https://n.whatwg.org/workz and https://n.whatwg.org/bla/work are also redirected with this. I guess that's harmless, but maybe `^/work$` or a `location = /work { return ... }` would be better?

Reply:
Yeah, this should be location most likely. We don't want to redirect all the things.

Reply:
>> https://n.whatwg.org/workz and https://n.whatwg.org/bla/work are also redirected with this. I guess that's harmless, but maybe `^/work$` or a `location = /work { return ... }` would be better?
>
> Yeah, this should be location most likely. We don't want to redirect all the things.

OK, fixed.